docs/content.zh/docs/dev/datastream/fault-tolerance/checkpointing.md

```diff
@@ -43,34 +43,40 @@ Flink 的 checkpoint 机制会和持久化存储进行交互，读写流与状
 
 ## 开启与配置 Checkpoint
```

Comment:
```suggestion
<a name="enabling-and-configuring-checkpointing"></a>

## 开启与配置 Checkpoint
```

It would be better to keep corresponding link tag for section subheading in the rest of the page.